content/en/docs/rate-limits.md

```diff
@@ -3,131 +3,140 @@ title: Rate Limits
-there's no limit on the number of distinct registered domains for which you can issue.
-So long as most of your customers don't have more than 2,000 subdomains on a
-registered domain, you most likely do not need an increase. See our [Integration Guide](/docs/integration-guide) for more advice.
-
-# <a id="clearing-pending"></a>Clearing Pending Authorizations
-
-If you have a large number of pending authorization objects and are getting a
-Pending Authorizations rate limiting error, you can trigger a validation attempt for those
-authorization objects by submitting a JWS-signed POST to one of its challenges, as
-described in the
-[ACME spec](https://tools.ietf.org/html/rfc8555#section-7.5.1).
-The pending authorization objects are represented by URLs of the form
-`https://acme-v02.api.letsencrypt.org/acme/authz/XYZ`, and should show up in your
-client logs. Note that it doesn't matter whether validation succeeds or fails.
-Either will take the authorization out of 'pending' state. If you do not
-have logs containing the relevant authorization URLs, you need to wait for the
-rate limit to expire. As described above, there is a sliding window, so this may
-take less than a week depending on your pattern of issuance.
-
-Note that having a large number of pending authorizations is generally the
-result of a buggy client. If you're hitting this rate limit frequently you
-should double-check your client code.
+# Foreword
+
+To ensure fair usage, Let's Encrypt enforces rate limits that we believe to be
+reasonable for most scenarios. Our rate limits were designed to facilitate unhampered
+certificate renewals and progressive certificate usage by large organizations. If you
+are integrating Let's Encrypt into your operations, please review our [Integration Guide](/docs/integration-guide)
+to ensure smooth operation. In particular, we recommend using one ACME account for many
+customers. If you are developing an ACME client or debugging the usage of one, please
+take advantage of our [staging environment](/docs/staging-environment), which is
+governed by higher rate limits and issues false certificates.
+
+We have no way of temporarily overriding the rate limits. For very large providers who are
+encountering rate limits due to sheer volume, we provide a [rate limiting form](https://goo.gl/forms/plqRgFVnZbdGhE9n1)
+that can be used to request higher rate limits. Since these requests can take several weeks
+to process, the form is not suitable for addressing temporary situations that will
+automatically be resolved when any encountered rate limits no longer apply.
+
+Revoked certificates still count against our rate limits and remain in the public record.
+All issued certificates can be found using [crt.sh](https://crt.sh/), which uses public
+[Certificate Transparency](https://www.certificate-transparency.org) logs. In general,
+certificate revocation should only be considered under extreme circumstances, such as the
```

Comment:
Maybe ‘abnormal’ or ‘non-routine’ rather than ‘extreme’?

Reply:
maybe "very special"? 🤔

I liken certificate revocations to exorcisms... oft-talked about with lack of correct knowledge and exceedingly-rarely applied under appropriate circumstances in practical reality.

Reply:
changed "extreme" to "very particular"